src/main/resources/config/application-artemis.yml

```diff
@@ -76,6 +76,20 @@ artemis:
         # Jenkins: The token value you use for the Server Notification Plugin and is stored under the notification-token credential above
         artemis-authentication-token-value: <token>
         build-timeout: 30 # Does cancel jenkins builds after 30 minutes to remove build that get stuck
+        # Defines the used docker images for certain programming languages
```

Comment:
Merge should have deleted these entries. Has been moved to `application.yml`

Reply:
Have now been removed from `application-artemis.yml` 👍